lib/yamel.ex

```diff
@@ -6,13 +6,25 @@ defmodule Yamel do
   @type t :: map() | list()
   @type yaml :: binary()
 
-  @spec decode!(yaml()) :: Yamel.t()
-  defdelegate decode!(yaml_string), to: YamlElixir, as: :read_from_string!
+  @spec decode!(yaml(), atoms: boolean()) :: Yamel.t()
+  def decode!(yaml_string, options \\ []) do
+    atoms = Keyword.get(options, :atoms)
```

Comment:
@the-pat I'll just ask you to use`keys: :atoms` as this options. The reason for that is so the interface of `Yamel` is as close as possible to [`Jason`](https://github.com/michalmuskala/jason).

Reply:
No problem! I'll hop on that.